test/runtime/fqdn.go

```diff
@@ -245,4 +304,67 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 		res := vm.ContainerExec(helpers.App1, helpers.CurlWithHTTPCode(allowedTarget))
 		res.ExpectSuccess("Cannot access to world1.cilium.test")
 	})
+
+	It("Interaction with other ToCIDR rules", func() {
+		policy := `
+[
+	{
+		"labels": [{
+			"key": "FQDN test - interaction with other toCIDRSet rules, no poller"
+		}],
+		"endpointSelector": {
+			"matchLabels": {
+				"container:id.app1": ""
+			}
+		},
+		"egress": [
+			{
+				"toPorts": [{
+					"ports":[{"port": "53", "protocol": "ANY"}],
+					"rules": {
+						"dns": [
+							{"matchPattern": "*.cilium.test"}
+						]
+					}
+				}]
+			},
+			{
+				"toFQDNs": [{
+					"matchName": "cilium.test"
```

Comment:
This is whitelisting `cilium.test` but we test to `world1.cilium.test`

Reply:
sorry, fix this in the next commit: 
https://github.com/cilium/cilium/pull/6298/commits/fcfd2fe5c97be1d0669f4b80da06326cb7109223

Will fix now.